CsvParser/csvParser.go

```diff
@@ -98,8 +97,8 @@ func CsvParser(filePath string, numIncidents ...int) []incident.DefaultIncident
 	return defaultIncidents
 }
 
-//convert a default incident that implements our interface to an incident object designed to exist in a json file
-//It returns incident of type IncidentData to 
+// Converts a default incident that implements our interface to an incident object designed to exist in a json file
```

Comment:
Thanks for making your changed comments consistent! Can you also change the other comments in this file? Please also remove the comment about assuming this returns 100 incidents